docs/en/status-communities/transfer-your-community-s-ownership.md

```diff
@@ -8,9 +8,36 @@ hide:
 
 # Transfer your community's ownership
 
- :octicons-tools-24: In progress
+:octicons-beaker-24: Beta
 
-!!! note ""
-     We're working on this content.
+You can transfer your Status Community ownership if you can't manage it and want to pass on the responsibility to someone else. To transfer your community's ownership, you need a [backup of the community's private key][back-up-your-community-s-private-key].
+
+When you transfer the ownership, the new owner's computer becomes a community node with full control of the community, including its settings, permissions and membership approvals.
+
+!!! tip
+    If you need help managing your community, consider adding others as admins. For more information, check out [Co-manage your Status Community][co-manage-your-status-community].
+
+## Security problems with the existing ownership model
```

Comment:
'Current security issues you should consider'? I think a title that directly addresses the user and sounds less intimidating and technical-ish implies a better chance of people at least scanning the section.